mora/service/cpr.py

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (c) 2017-2018, Magenta ApS
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+import flask
+
+from mora.integrations.serviceplatformen import get_citizen
+from mora.service import keys
+
+blueprint = flask.Blueprint('cpr', __name__, static_url_path='',
+                            url_prefix='/service')
+
+
+@blueprint.route('/e/cpr_lookup/<string:cpr>/')
```

Comment:
This feels vaguely inconsistent — perhaps something like `/e/create` or `/e/import`?

Reply:
This isn't a creation or import, but rather just a lookup in SP on a CPR. The resulting data is not written to LoRa, but returned to the UI.

Reply:
Please clarify that in the docstring — along with documenting the response while at it.

We should probably convert the path parameter to a query parameter — that's more customary for searches — and please and make it use `util.restrictargs()` too.